src/FINDOLOGIC/Helpers/ResponseHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace FINDOLOGIC\Helpers;
+
+class ResponseHelper
+{
+    /**
+     * Gets a property from an object.
+     *
+     * @param mixed $obj
+     * @param string $property
+     * @param null|string $type Convert value to another type. Optional.
+     * @param bool $emptyValueIsAllowed If true, values that are 0 or 0.0 are allowed.
+     *
+     * @return mixed|null Returns the property value or null if it does not exist.
+     */
+    public static function getProperty($obj, $property, $type = null, $emptyValueIsAllowed = false)
+    {
+        if (!property_exists($obj, $property)) {
```

Comment:
Is it wise to just return `null` if something doesn't exist? Something being missing could be indicative of a bug. For truly optional things, perhaps having a way to provide a default value would be an improvement.

Reply:
Why would that be an improvement? How can something exist in my API Wrapper that does not exist in the actual XML response?  
Let's say I want to get the color of an `Item`, this means I need to call `$item->getColor()`. What do I expect an empty string or `null`, which tells the consumer "Hey I do not have a color"?  

I am not sure, but maybe the best way is to make this configurable via the FINDOLOGIC API Config? But I am up for discussion ;).